components/Hero.tsx

```diff
@@ -1,14 +1,20 @@
 import React from "react";
 import styled from "@emotion/styled";
+import { useRouter } from "next/router";
 import { MaxWidthWrapper } from "./Wrapper";
 import { SearchBar } from "./SearchBar";
 import { QUERIES } from "../utils/constants";
 
-export const Hero: React.FC = ({ children }) => {
+type HeroProps = {
+  topAction?: React.ReactNode;
+};
```

Comment:
What is this actually doing? I'm unclear what topAction is. Maybe document this or rename it?

Reply:
This lets you add an action on top of the Search bar in the Hero, like `Go back to Projects`, it is useful when you want to reuse the same component but need a slightly different UI. 
I'm very open to better naming / pattern for this sort of thing though. This is just the first way that came to mind